src/resample.c

```diff
@@ -74,6 +75,48 @@ main (int argc, char *argv [])
 		{	k ++ ;
 			converter = parse_int_or_die (argv [k], "converter") ;
 			}
+		else if (strcmp (argv [k], "-r") == 0)
+		{   k ++ ;
+            if (sscanf (argv [k], "%d,%d,%c,%d", &sfinfo.samplerate, &sfinfo.channels, &raw_type, &raw_bits) != 4)
+            {   printf ("Error : unable to decode raw audio parameters.\n") ;
+                usage_exit (argv [0]) ;
+                }
+            sfinfo.format = SF_FORMAT_RAW;
+            if (raw_type == 'i' && ( raw_bits == 8 || raw_bits == 16 || raw_bits == 24 || raw_bits == 32 ) )
```

Comment:
I am seeing some new code for handling command line args, but no update to the help output. I think a help output update is needed.

Reply:
what about https://github.com/eblanca/sndfile-tools/blob/1375f19ee95e1f7d362226ff651c5a4fe6681307/src/resample.c#L344 ?